crates/services/gas_price_service/src/fuel_gas_price_updater/fuel_core_storage_adapter/metadata_tests.rs

```diff
@@ -36,23 +40,30 @@ fn database() -> StorageTransaction<InMemoryStorage<GasPriceColumn>> {
     InMemoryStorage::default().into_transaction()
 }
```

Comment:
hey @MitchTurner do you know why we return a `StorageTransaction` here?

Reply:
IIRC just because it met the requirements and was simple to work with. I don't think it's particularly important or like we're attached to it.

`Database` itself is a `fuel-core` concept, so we don't have access to it from `gas_price_service`.